AutomatedTesting/Gem/PythonTests/editor/EditorScripts/Sample_Editor_Tests.py

```diff
@@ -0,0 +1,143 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Test Case: Creating tests for basic editor feature like creating a level, adding new entities, modifying the entities etc.
```

Comment:
This can be removed, there's a description in the body of the test function.

Reply:
fixed